### DISCUSSION TITLE
Export HTML functionality is missing in new versions

### DISCUSSION THREAD

**Discussion by max-arshinov** [structurizr/lite#75] [Q&A] [answered]
### Question

In previous versions of Structurizr Lite, it was possible to load an HTML file with all diagrams from the workspace. It seems that this option has been removed. I used this feature in the past to quickly send intermediate results to business users not familiar with docker. Should I publish workspaces to StructurizR On-Prem instead? Also, I am wondering why this feature was removed.

**Comment by simonbrowndotje** [comment, marked as answer]
See https://github.com/structurizr/lite/discussions/34 ... this feature likely won't be making a return.
